admin/sql/CreateFunctions.sql

```diff
@@ -1100,6 +1100,18 @@ BEGIN
 END;
 $$ LANGUAGE 'plpgsql';
 
+CREATE OR REPLACE FUNCTION unique_primary_genre_alias()
```

Comment:
Now you need a corresponding `unique_primary_for_locale` trigger that calls this function.

Reply:
Added. To -fks only, since it seems to have been done like that before, but do confirm.